saleor/webhook/observability/sensitive_data.py

```diff
@@ -1,23 +1,47 @@
 """Definitions of sensitive data for observability obfuscation methods.
 
-SENSITIVE_HEADERS is a tuple of sensitive HTTP headers to anonymize before reporting.
+ALLOWED_HEADERS is a set of lowercase HTTP headers allowed for reporting.
+
+SENSITIVE_HEADERS is a set of lowercase HTTP headers to anonymize before reporting.
 
 SENSITIVE_GQL_FIELDS is a dict of sets representing fields of GraphGL types to anonymize
 - Type
 - Fields
 """
 from typing import Dict, Set
 
-from ...core.auth import DEFAULT_AUTH_HEADER, SALEOR_AUTH_HEADER
+from ...app.headers import AppHeaders, DeprecatedAppHeaders
 
-SENSITIVE_HEADERS = (
-    SALEOR_AUTH_HEADER,
-    DEFAULT_AUTH_HEADER,
-    "COOKIE",
-)
-SENSITIVE_HEADERS = tuple(
-    x[5:] if x.startswith("HTTP_") else x for x in SENSITIVE_HEADERS
-)
+ALLOWED_HEADERS = {
+    header.lower()
+    for header in {
+        "Content-Length",
+        "Content-Type",
+        "Host",
+        "Origin",
+        "Referer",
+        "Content-Encoding",
+        "User-Agent",
+        "Cookie",
+        "Authorization",
+        "Authorization-Bearer",
```

Comment:
Is there actually a use-case for forwarding those whatsoever? (I know they are getting masked with `***`)

Reply:
The main use-case for it is debugging - as those are headers used by Saleor, I wanted to explicitly show that they are sent to or from Saleor and not confuse observability users (e.g., apps creators)